pkg/controller/manager/manager_controller.go

```diff
@@ -264,183 +289,208 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 	if err != nil && !errors.IsNotFound(err) {
-		r.status.SetDegraded(operatorv1.ResourceReadError, "Error while fetching Authentication", err, reqLogger)
+		r.status.SetDegraded(operatorv1.ResourceReadError, "Error while fetching Authentication", err, logc)
 		return reconcile.Result{}, err
 	}
 	if authenticationCR != nil && authenticationCR.Status.State != operatorv1.TigeraStatusReady {
-		r.status.SetDegraded(operatorv1.ResourceNotReady, fmt.Sprintf("Authentication is not ready authenticationCR status: %s", authenticationCR.Status.State), nil, reqLogger)
+		r.status.SetDegraded(operatorv1.ResourceNotReady, fmt.Sprintf("Authentication is not ready authenticationCR status: %s", authenticationCR.Status.State), nil, logc)
 		return reconcile.Result{}, nil
 	} else if authenticationCR != nil {
-		trustedSecretNames = append(trustedSecretNames, render.DexTLSSecretName)
+		// trustedSecretNames = append(trustedSecretNames, render.DexTLSSecretName)
 	}
 
+	// TODO: Trusted bundle for all components will be in the same namespace for multi-tenancy.
+	// So, we'll need to refactor this. I think for multi-tenancy, we can simplify the trusted-bundle generation
+	// altogether so that we only ever need a single cert for it. In single-tenant, we need more complexity in order
+	// to support BYO certs.
+	//
+	// That said, it's probably a good idea to move certificate management to its own controller anyway so that
+	// it's not so scattered!
 	trustedBundle := certificateManager.CreateTrustedBundle()
-	for _, secretName := range trustedSecretNames {
-		certificate, err := certificateManager.GetCertificate(r.client, secretName, common.OperatorNamespace())
+	for _, secret := range trustedSecretNames {
+		certificate, err := certificateManager.GetCertificate(r.client, secret.Name(), secret.Namespace())
 		if err != nil {
-			r.status.SetDegraded(operatorv1.CertificateError, fmt.Sprintf("Failed to retrieve %s", secretName), err, reqLogger)
+			r.status.SetDegraded(operatorv1.CertificateError, fmt.Sprintf("Failed to retrieve %s", secret), err, logc)
 			return reconcile.Result{}, err
 		} else if certificate == nil {
-			reqLogger.Info(fmt.Sprintf("Waiting for secret '%s' to become available", secretName))
-			r.status.SetDegraded(operatorv1.ResourceNotReady, fmt.Sprintf("Waiting for secret '%s' to become available", secretName), nil, reqLogger)
+			logc.Info(fmt.Sprintf("Waiting for secret '%s' to become available", secret))
+			r.status.SetDegraded(operatorv1.ResourceNotReady, fmt.Sprintf("Waiting for secret '%s' to become available", secret), nil, logc)
 			return reconcile.Result{}, nil
 		}
 		trustedBundle.AddCertificates(certificate)
 	}
-	certificateManager.AddToStatusManager(r.status, render.ManagerNamespace)
+	certificateManager.AddToStatusManager(r.status, request.InstallNamespace())
 
 	// Check that Prometheus is running
+	// TODO: We'll need to run an instance of Prometheus per-tenant? Or do we use labels to delimit metrics?
```

Comment:
We actually don't need to run Prometheus in the management cluster except for our own metrics. We don't store product data in the management cluster prometheus AFAIK